postgrest_py/_async/request_builder.py

```diff
@@ -36,10 +38,13 @@ async def execute(self) -> Tuple[Any, Optional[int]]:
             self.path,
             json=self.json,
         )
-        return process_response(self.session, r)
+        try:
+            return APIResponse.from_http_request_response(r)
+        except ValueError as e:
+            raise APIError(r.json()) from e
 
 
-class AsyncFilterRequestBuilder(BaseFilterRequestBuilder, AsyncQueryRequestBuilder):
+class AsyncFilterRequestBuilder(BaseFilterRequestBuilder, AsyncQueryRequestBuilder):  # type: ignore
```

Comment:
Why is necessary to ignore type checking?

Reply:
In this [commit](https://github.com/supabase-community/postgrest-py/commit/790811f00764b3fb1734bfdb25964972ed93a2ca) I tagged the mypy issue.

Reply:
Please, add the reference like a comment in the code.

Reply:
Just by writing the url next to type: ignore? @leynier